vscode-trace-extension/src/trace-explorer/available-views/trace-explorer-available-views-webview-provider.ts

```diff
@@ -54,7 +48,7 @@ export class TraceExplorerAvailableViewsProvider extends AbstractTraceExplorerPr
         if (data?.descriptor) {
             // FIXME: JSONBig.parse() created bigint if numbers are small.
```

Comment:
Left-over from previous implementation? (i.e. even if OutputDescriptor contained regular number field(s), normalization would fix it?)